### ISSUE TITLE
Accessibility: Narrator should read the position of Mat-Tab

### ISSUE THREAD

**Issue by paraskt** [angular/material2#11369]
#### Bug, feature request, or proposal:
Feature request

#### What is the expected behavior?
When using a mat-tab, the narrator should read the position of tab within tab group (i.e. tab item 2 of 4). 

#### What is the current behavior?
When using a mat-tab, the narrator does not read the position of tab within tab group (i.e. tab item 2 of 4). aria-posinset and aria-setsize could be used to achieve this in mat-tab code.

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>

#### What is the use-case or motivation for changing an existing behavior?
This greatly helps a visually impaired user's experience navigating throughout a product.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular version: 5.1.3
Browser: Chrome, Edge

**Comment by jelbourn** [comment]
I don't quite follow. What is the combination of mat-menu and mat-tab-group that you're referring to?

**Comment by paraskt** [comment]
Sorry, I typed mat-menu by mistake. What I meant was, narrator is not reading the position of mat-tab inside mat-tab-group.
I have updated my original comment.

**Comment by paraskt** [comment]
I am using windows Narrator, it comes bundled with windows.
OS: Windows 10
OS Version: 1803

**Comment by jelbourn** [comment]
Does the problem manifest with another reader? We don't specifically support Windows Narrator
(we support JAWS, NVDA, VoiceOver, ChromeVox on ChromeOS, and TalkBalk)

**Comment by paraskt** [comment]
I tested on NVDA, and the issue is present over there as well. It should be very straightforward to fix with aria-poinset and aria-setsize.

Thanks,
Paras
